packages/gen-ai/bff/internal/constants/configmap_templates.go

```diff
@@ -91,6 +81,5 @@ benchmarks: []
 tool_groups:
 - toolgroup_id: builtin::rag
   provider_id: rag-runtime
-external_providers_dir: /opt/app-root/.llama/providers.d
```

Comment:
@akram Have you tested this without external_providers_dir? traditionally removing it has caused us issues with the configmap

Reply:
that was with the old image pre- 0.3.0